src/Widgets/SpaceWidget/SpaceWidgetComponent.js

```diff
@@ -0,0 +1,7 @@
+import * as React from "react";
+import * as Scrivito from "scrivito";
+
+Scrivito.provideComponent("SpaceWidget", ({ widget }) => {
+  const size = widget.get("size") || 1;
```

Comment:
I think `5` is a better default, then `1`:

```suggestion
  const size = widget.get("size") || 5;
```